packages/graphql-codegen-cli/tests/generate-and-save.spec.ts

```diff
@@ -324,6 +309,7 @@ describe('generate-and-save', () => {
     });
 
     test('Document syntax error - should print native GraphQLError', async () => {
+      expect.assertions(4);
```

Comment:
`expect.assertions(4);` is needed here to avoid the scenario where `await generate` somehow passes, which means it'd run just the `try` block. `try` block doesn't have an expect so it'd pass the test.